ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -391,7 +391,9 @@ export class WalletDB {
     return meta
   }
 
-  async *loadAccounts(tx?: IDatabaseTransaction): AsyncGenerator<AccountValue, void, unknown> {
+  async *loadAccounts(
+    tx?: IDatabaseTransaction,
+  ): AsyncGenerator<DecryptedAccountValue, void, unknown> {
```

Comment:
Same comment above regarding the generic. Will update once encrypted account encoding is added.